pynxtools/dataconverter/readers/stm/README.md

```diff
@@ -0,0 +1,130 @@
+# STM/STS reader
+## Contact persion in FAIRmat for this reader
+**Rubel Mozumder (mozumder@physik.hu-berlin.de)**
+## Reader Notes:
+- Reader builds on NXsts
+- Can parse bias spectroscopy (STS) from
+    - Nanonis: Generic 5e, Generic 4.5
+- Can parse STM from
+    - Nanonis: Generic 5e, Generic 4.5
+
+## Some usages:
+- The data structure of the input data file can be investigate with the code below:
+    ```
+        from pynxtools.dataconverter.readers.stm import get_stm_raw_file_info
+        from pynxtools.dataconverter.readers.stm import get_sts_raw_file_info
+
+        # for stm (.sxm) file
+        get_stm_raw_file_info('STM_nanonis_generic_5e.sxm')
+
+        # for sts (.dat) file
+        get_sts_raw_file_info('STS_nanonis_generic_5e_1.dat')
+    ```
+- To run STM reaader use the folowing code
+    ```
+    # Run STM reader
+
+    !dataconverter \
+    --reader stm \
+    --nxdl NXsts \
+    --input-file STM_nanonis_generic_5e.sxm \
+    --input-file ../config_file_for_sxm.json \
+    --input-file ./Nanonis_Eln.yaml \
+    --output final_stm_dev_.nxs
+
+    ```
+- Run STS reader with
+    ```
+    # Run STS reader
+
+    !dataconverter \
+    --reader stm \
+    --nxdl NXsts \
+    --input-file ./STS_nanonis_generic_5e_1.dat \
+    --input-file ../config_file_for_dat.json \
+    --input-file Nanonis_Eln.yaml \
+    --output ./final_sts_dev.nxs
+    ```
+
+- Utilization of ELN
```

Comment:
I do not have any plan to keep the example files in Pynxtools. Because they are already in nomad-remote-tools-hub and both nomad-remote-tool-hub and pynxtools both are submodules in  nomad.